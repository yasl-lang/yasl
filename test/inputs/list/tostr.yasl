echo [
    {
        .accountName: .root,
        .nickname: .Cumulus,
        .peerId: .QmWV77RTC7cwMPbPBfCbP68JGPt5ta8qPyiCWsUNZsXvpo,
        .groups: ['パン:0'] },
    {
        .accountName: .root,
        .nickname: .Myself,
        .peerId: .QmemJHsMDuBjUCAyiWeVdct2LGHqtZhu8QQCt8ZQVbY1qz,
        .groups: ['パン:0'] }]

# check that we correctly forward the specifier to each element.
echo [ 16, 32, 10, 45 ]->tostr('x')
