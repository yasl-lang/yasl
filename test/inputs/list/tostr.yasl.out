[{accountName: root, nickname: Cumulus, peerId: QmWV77RTC7cwMPbPBfCbP68JGPt5ta8qPyiCWsUNZsXvpo, groups: [パン:0]}, {accountName: root, nickname: Myself, peerId: QmemJHsMDuBjUCAyiWeVdct2LGHqtZhu8QQCt8ZQVbY1qz, groups: [パン:0]}]
[0x10, 0x20, 0xa, 0x2d]
